src/index.ts

```diff
@@ -76,7 +86,7 @@ function main(): void {
     .option('no-input', {
       boolean: true,
       coerce: once(processNoInput),
-      default: false,
+      default: isCI,
```

Comment:
Do I understand correctly (from [this comment](https://github.com/getsentry/craft/pull/203#issuecomment-824254467)) that this is intended to work around the `esbuild` warning with `update-notifier` lazy-loading? If so, what happens if I run `esbuild`-built `craft` outside of CI? 🤔

At the very least we should update the README/usage docs to reflect what we're doing here.

Reply:
> If so, what happens if I run esbuild-built craft outside of CI? 🤔

Don't see how this is relevant. I now made `is-ci` an explicit required dependency of the project. We use it to auto detect/set the value of `--no-input` argument. Not sure if it needs a comment as if somebody removes this require/usage, builds will start failing.

Reply:
Right, but why are we doing that on this PR? Is it related somehow to `update-notifier`?

Reply:
Yes, without us requiring this library explicitly, `update-notifier` fails as it lazily imports this library and since nothing else refers to it, it is not bundeled.

Reply:
Ah, gotcha gotcha, so `update-notifier` requires this, and since we now need to explicitly require it we figured why not make use of it elsewhere, something like that?

Reply:
>Ah, gotcha gotcha, so update-notifier requires this, and since we now need to explicitly require it we figured why not make use of it elsewhere, something like that?

Yup, but turns out even that doesn't work so 🤷🏻‍♂️